servers/rendering/renderer_rd/shaders/sdfgi_direct_light.glsl

```diff
@@ -20,10 +20,10 @@ layout(set = 0, binding = 3, std430) restrict readonly buffer DispatchData {
 dispatch_data;
 
 struct ProcessVoxel {
-	uint position; //xyz 7 bit packed, extra 11 bits for neighbors
-	uint albedo; //rgb bits 0-15 albedo, bits 16-21 are normal bits (set if geometry exists toward that side), extra 11 bits for neibhbours
-	uint light; //rgbe8985 encoded total saved light, extra 2 bits for neighbours
-	uint light_aniso; //55555 light anisotropy, extra 2 bits for neighbours
+	uint position; // xyz 7 bit packed, extra 11 bits for neighbors.
+	uint albedo; // rgb bits 0-15 albedo, bits 16-21 are normal bits (set if geometry exists toward that side), extra 11 bits for neibhbors.
```

Comment:
_nei**g**hbors_ still appears to be misspelt _nei**b**hbors_

Reply:
Thanks, fixed in: 02b6eed